src/components/AnimeGridList.js

```diff
@@ -12,10 +12,6 @@ const GridContainer = Styled.section`
 `;
 
 class AnimeGridList extends React.PureComponent {
-  static propTypes = {
-    animes: PropTypes.arrayOf(PropTypes.object).isRequired,
-  };
-
```

Comment:
me parece que esta repetido 🤔

Reply:
@carlosfcocruz - Good catch, Addressed on this commit: 1882aa18d719c5ee97ccf98c68076b9f160804f3

Thanks 👍